src/java.base/share/classes/sun/security/ssl/SSLSessionImpl.java

```diff
@@ -73,7 +72,7 @@
  *
  * @author David Brownell
  */
-final class SSLSessionImpl extends ExtendedSSLSession {
+final public class SSLSessionImpl extends ExtendedSSLSession {
```

Comment:
Use blessed modifiers order
```suggestion
public final class SSLSessionImpl extends ExtendedSSLSession {
```

Reply:
Actually being public wasn't necessary anymore.